tests/tests_integration/test_api/test_datapoint_subscriptions.py

```diff
@@ -20,7 +21,9 @@
 )
 from cognite.client.utils._text import random_string
 
-TIMESERIES_EXTERNAL_IDS = [f"PYSDK DataPoint Subscription Test {no}" for no in range(20)]
+TIMESERIES_EXTERNAL_IDS = [
+    f"PYSDK DataPoint Subscription {platform.system()} {platform.python_version()} Test {no}" for no in range(20)
+]
```

Comment:
May I suggest we make this a fixture?
```suggestion
def time_series_external_ids(os_and_py_version: str) -> list[str]:
    return [
        f"PYSDK DataPoint Subscription {os_and_py_version} Test {no}" for no in range(20)
    ]
```

Reply:
Sure can do that. What is the main motivation as it is just constants anyways?

Reply:
I agree it is not necessary, but you get `os_an_py_version` for free which is nice 🤓